build/tasks.js

```diff
@@ -67,7 +67,7 @@ const Tasks = {
 
   async serve() {
     // Make sure we'll run in development mode to enable logging.
-    await Lazy.config({ development: true });
+    await Lazy.config({ development: true, offerDefault: false });
```

Comment:
We should add `offerDefault: false` in `build/utils/base-config.js` and only offerDefault: true in `task-package.js`